lambdas/service/app.py

```diff
@@ -981,7 +992,8 @@ def update_cart(cart_id):
     :return: The updated cart
         {
             "CartName": str,
-            "CartId": str
+            "CartId": str,
+            "DefaultCart": str
```

Comment:
I see this typed as `int` in other place. This and the fact that we use a `int` as a `bool` makes this variable a bit ambiguous. Should it be named `IsDefaultCart`?

Reply:
By the way, should the field name (in DynamoDB) remain as "DefaultCart" or do you want me to rename it to `IsDefaultCart` too?